content/neuvector-prime/aws/contents.lr

```diff
@@ -0,0 +1,164 @@
+title: NeuVector Prime in AWS: Usage Instructions
+---
+theme: suse
+---
+body:
+
+## Prerequisites
+
+1. These instructions assume you have an EKS cluster installed and that you have access to that cluster via a `kubeconfig`.
+2. Installation requires you have the following tools available and properly configured to access your AWS account, and your EKS cluster:
+* `aws`
+* `curl`
+* `eksctl`
+* `helm` (v3 or greater)
+
+## Preparing your cluster
+
+### OIDC provider
+
+Your EKS cluster is required to have an OIDC provider installed. To check for an OIDC provider first find the OIDC issuer with the following command. Substitute `$CLUSTER_NAME` with the *Name* of the EKS cluster:
+
+```
+aws eks describe-cluster --name $CLUSTER_NAME --query cluster.identity.oidc.issuer --output text
+```
+
+A URL is returned, like `https://oidc.eks.region.amazonaws.com/id/1234567890ABCDEF`. The part after `https://` will be referred to in later instructions as the *OIDC Provider Identity*. The final section of the URL, `1234567890ABCDEF`, is the $OIDC_ID.
+
```

Comment:
I think it is helpful to explicitly add the identity, albeit it being somewhat repetitive.

s|Identity*.|Identity*, i.e. oidc.eks.region.amazonaws.com/id/1234567890ABCDEF|

this also eliminates potential confusion about whether whether or not everything is the identity or only parts of it, i.e. to the first separator `/`/

Reply:
Added in [33bcbb5](https://github.com/SUSE-Enceladus/marketplace-docs/pull/14/commits/33bcbb5a2051fdfe1f229cfb0daa50c5c1423086).